projects/core/src/occ/adapters/configurator/variant/occ-configurator-variant.adapter.ts

```diff
@@ -63,6 +65,32 @@ export class OccConfiguratorVariantAdapter
       .pipe(this.converterService.pipeable(CONFIGURATION_NORMALIZER));
   }
 
+  addToCart(
+    userId: string,
+    cartId: string,
+    productCode: string,
+    quantity: number = 1,
+    configId: string
+  ): Observable<CartModification> {
+    const url = this.occEndpointsService.getUrl('addConfigurationToCart', {
+      userId,
+      cartId,
+    });
+
+    const occConfigOrderEntry = JSON.stringify({
```

Comment:
Call Serializer (needs to be created) instead

Reply:
done